ai-decision-services/ai-ad-dpp-engine/introduction/introduction.md

```diff
@@ -2,46 +2,78 @@
 
 ## About this Workshop
 
-This introduction covers the complete "parent" workshop. Use this text to set up the story for the workshop. Be engaging - what will the learner get from spending their time on this workshop?
+In this lab, we are going to offer a detailed hands-on onboarding guide for users who are interested in our automated end to end **Anomaly Detection** data preprocessing and training/inference for workflow.
 
-Estimated Workshop Time: -- hours -- minutes (This estimate is for the entire workshop - it is the sum of the estimates provided for each of the labs included in the workshop.)
+## Objectives
+This solution will allow you to configure a training pipeline and an inference pipeline in your tenancy, along with ingestion and preprocessing of your data. Input sizes could range from a few rows to several tens of millions of rows, and 1 column to several thousands of columns.
 
-*You may add an option video, using this format: [](youtube:YouTube video id)*
 
-  [](youtube:zNKxJjkq0Pw)
+## Workflow introduction
 
-### Objectives
+![workflow](./images/workflow.png)
 
-*List objectives for the workshop*
+A brief description of the workflow:
 
-In this workshop, you will learn how to:
+1.  Prepare the driver config and upload it to the config bucket. Also create an anomaly detection (AD) application for later usage.
+2.  Upload a training/inferencing dataset to the input source bucket.
+3.  The upload event will trigger an event to the OCI event listening to this bucket.
+4.  OCI event will trigger the downstream Orchestrator deployed on OCI Function, which will load the driver config and start the workflow.
+5.  The workflow will run based on the driver config.
+6.  Once the process is successful, the processed data with other informations (for example, model\_info) will be written to the staging bucket.
```

Comment:
Consider rewording - As soon as the data processing workflow completes successfully, the processed data ....